<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMPSettingsRespawn</class>
 <widget class="QWidget" name="PageMPSettingsRespawn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="10" column="1">
    <widget class="QSpinBox" name="fieldLoadoutCamTime">
     <property name="toolTip">
      <string>Control how much time players spend on the initial loadout camera. If this is too short, players won't have a chance to pick their loadouts before spawning! The default is ten seconds.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelSuicidePenalty">
     <property name="text">
      <string>Suicide Penalty</string>
     </property>
     <property name="buddy">
      <cstring>fieldSuicidePenalty</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelLivesPerRound">
     <property name="text">
      <string>Lives per Round</string>
     </property>
     <property name="buddy">
      <cstring>fieldLivesPerRound</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="fieldSuicidePenalty">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelLoadoutCamTime">
     <property name="text">
      <string>Loadout Camera Time</string>
     </property>
     <property name="buddy">
      <cstring>fieldLoadoutCamTime</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="fieldLivesPerRound">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelRespawnTime">
     <property name="text">
      <string>Respawn Time</string>
     </property>
     <property name="buddy">
      <cstring>fieldRespawnTime</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelTeamLivesPerRound">
     <property name="text">
      <string>Team Lives per Round</string>
     </property>
     <property name="buddy">
      <cstring>fieldTeamLivesPerRound</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelBetrayalPenalty">
     <property name="text">
      <string>Betrayal Penalty</string>
     </property>
     <property name="buddy">
      <cstring>fieldBetrayalPenalty</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="fieldRespawnTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="fieldTraitsDuration">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelRespawnGrowth">
     <property name="text">
      <string>Respawn Time Growth</string>
     </property>
     <property name="buddy">
      <cstring>fieldRespawnGrowth</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="labelTraitsDuration">
     <property name="text">
      <string>Respawn Traits Duration</string>
     </property>
     <property name="buddy">
      <cstring>fieldTraitsDuration</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="fieldRespawnGrowth">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="fieldBetrayalPenalty">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="fieldTeamLivesPerRound">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldRespawnWithTeammate">
     <property name="text">
      <string>Respawn with Teammate (unverified)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldSyncWithTeam">
     <property name="text">
      <string>Synchronize with Team</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldRespawnAtLocation">
     <property name="text">
      <string>Respawn at Location (unverified)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldRespawnOnKills">
     <property name="text">
      <string>Respawn on Kills</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
